### ISSUE TITLE
showing the menu dots and/or icons only on hover does not work well on mobile

### ISSUE THREAD

**Issue by szaimen** [nextcloud/nextcloud-vue#3524]
- showing the menu dots and/or icons only on hover does not work well on mobile

![image](https://user-images.githubusercontent.com/42591237/202497612-25d80bf5-3e3e-4e6f-bf25-2c212f375b95.png)

_Originally posted by @szaimen in https://github.com/nextcloud/server/pull/35080#pullrequestreview-1184615215_

**Comment by szaimen** [comment]
cc @artonge @jancborchardt @marcoambrosini @nimishavijay

**Comment by jancborchardt** [comment]
For the versions list, it would be fine to permanently show the 3-dot icon for all entries.

This is mainly done this way for the navigation because:
- It is always visible so way more visual noise
- The navigation entries can have an active state in whicj the menu icon can be displayed

**Comment by szaimen** [comment]
Cc @marcoambrosini could you give me a pointwr how to allow to do this? Thanks a lot!

**Comment by marcoambrosini** [comment]
<img width="919" alt="Screenshot 2022-11-30 at 10 46 44" src="https://user-images.githubusercontent.com/26852655/204763021-fd679564-71a2-48ff-846f-73c4bb8a4c66.png">

:D

**Comment by szaimen** [comment]
Cool! So this already exists :)
